### PR TITLE
Migrate tests from TL2.fund() to FTLA.fund() in TrueFiPool2.test.ts

### PR DESCRIPTION
**Pull request by skalermo**

### CODE REVIEW

**Comment by skalermo** [comment]
I'm aware that SAFU tests are failing, those are handled in #1010. Once #1010 is merged this PR should be ready to go.

File: contracts/truefi2/FixedTermLoanAgency.sol [hunk 1]
```diff
@@ -226,6 +226,10 @@ contract FixedTermLoanAgency is IFixedTermLoanAgency, UpgradeableClaimable {
         emit BorrowingMutexChanged(newMutex);
     }
 
+    function setRateAdjuster(ITrueRateAdjuster newRateAdjuster) public onlyOwner {
+        rateAdjuster = newRateAdjuster;
+    }
```
Comment (vanruch) [inline contracts/truefi2/FixedTermLoanAgency.sol:231]: Can we avoid adding unnecessary setters to contracts?

Comment (yuchenlintt) [reply contracts/truefi2/FixedTermLoanAgency.sol:231]: +1.

If we're launching the first version with TrueRateAdjuster (=> CreditModel) in the initializer, then we won't need to add a setter.

File: test/truefi2/TrueFiPool2.test.ts [hunk 2]
```diff
@@ -763,49 +737,31 @@ describe('TrueFiPool2', () => {
...
-      )
 
-      await expect(lender.connect(borrower).fund(loan1.address))
-        .to.be.revertedWith('TrueFiPool: Insufficient liquidity')
+      const amount = (await tusdPool.poolValue()).div(10).add(1)
+      const mockRateAdjuster = await deployMockContract(owner, TrueRateAdjusterJson.abi)
+      await mockRateAdjuster.mock.fixedTermLoanAdjustment.returns(0)
+      await mockRateAdjuster.mock.rate.returns(0)
+      await mockRateAdjuster.mock.borrowLimit.returns(parseEth(100_000_000))
+      await ftlAgency.setRateAdjuster(mockRateAdjuster.address)
```
Comment (yuchenlintt) [inline test/truefi2/TrueFiPool2.test.ts:746]: Ah, I see what setRateAdjuster was for.

For tests like this, would it work to pass this mockRateAdjuster in to setupTrueFi2 as an argument?

**Review by yuchenlintt — CHANGES_REQUESTED** [review]
I'll merge 1010 in, and then let's see if we can avoid adding a `setRateAdjuster()` function.

**Review by yuchenlintt — APPROVED** [review]

File: test/truefi2/SAFU.test.ts [hunk 3]
```diff
@@ -1,5 +1,16 @@
...
+  createLoan,
+  DAY,
+  parseTRU,
+  parseUSDC,
+  parseEth,
+  setupTruefi2,
+  timeTravel as _timeTravel,
+  createDebtToken as _createDebtToken,
+  extractLoanTokenAddress,
+} from 'utils'
```
Comment (MiksuJak) [inline test/truefi2/SAFU.test.ts:13]: image

**Review by vanruch — APPROVED** [review]

**Pull request merged**